controllers/TuitController.ts

```diff
@@ -121,6 +126,27 @@ export default class TuitController implements ITuitController {
     TuitController.tuitDao
       .updateTuit(req.params.tid, req.body)
       .then((status) => res.send(status));
+  
+  /**
+   * @param {Request} req Represents request from client, including path
+   * parameter tid identifying the primary key of the tuit to be modified
+   * @param {Response} res Represents response to client, including status
+   * on whether updating a tuit was successful or not
+   */
+   editTuit = async (req: Request, res: Response) => {
```

Comment:
As we talked about over the meeting yesterday, it would be better to have only the `tuit` parameter in the request body because we want the user to only be able to edit the text of the tuit and not any other parts. 
So we would get the `tid` from the URL param and `tuit` (text) from the request body.

Reply:
Makes sense, will update change shortly!